crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -10,27 +10,30 @@ use std::path::PathBuf;
 pub(crate) struct TestContractCommand {
 	#[arg(short = 'p', long, help = "Path for the contract project [default: current directory]")]
 	path: Option<PathBuf>,
-	#[arg(short = 'f', long = "features", help = "Features for the contract project")]
+	#[arg(short = 'f', long = "features", help = "Features for the contract project", value_parser=["e2e-tests"])]
 	features: Option<String>,
+	#[arg(
+		short = 'n',
+		long = "contracts-node",
+		help = "Path to the contracts node to run e2e tests [default: none]"
+	)]
+	contracts_node_path: Option<PathBuf>,
```

Comment:
Its obvious its a path based on the type. Suggest just `node`

Reply:
Same as previous comment, renamed.